web/blog/2021-12-02-waspello.md

```diff
@@ -169,14 +170,7 @@ This is pretty much it from the stuff that works 😄 ! I kinda rushed a bit thr
 The main problem of the current implementation of Waspello is the **lack of support for optimistic UI updates in Wasp**. What this means is that currently, when an entity-related change is made (e.g. a card is moved from one list to another), we have to wait until that change is fully executed on the server until it is visible in the UI, which causes a noticeable delay.  
 In many cases that is not an issue, but when UI elements are all visible at once and it is expected from them to be updated immediately, then it is noticeable. This is also one of the main reasons why we chose to work on Waspello - to have a benchmark/sandbox for this feature! Due to this issue, here's how things currently look like:
 
-<p align="center">
-  <figure>
-    <img alt="Waspello - no optimistic UI update"
-        src={useBaseUrl('img/waspello-no-opt-UI-updates.gif')}
-    />
-    <figcaption class="image-caption">Without an optimistic UI update, there is a delay</figcaption>
-  </figure>
-</p>
+<ImgCap alt="Waspello - no optimistic UI update" source="img/waspello-no-opt-UI-updates.gif" caption="Without an optimistic UI update, there is a delay"/>
```

Comment:
```suggestion
<ImgWithCaption alt="Waspello - no optimistic UI update" source="img/waspello-no-opt-UI-updates.gif" caption="Without an optimistic UI update, there is a delay"/>
```
I would propose a tad more explanatory name, sth like this

Reply:
Okay, i will do this